### PR TITLE
Fix stack lint naming in `lint` CI action

### PR DESCRIPTION
**Pull request by mjgpy3**
**Why?**

The project was recently renamed.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -47,25 +47,36 @@ jobs:
...
       - id: prep
         run: |
           printf "::set-output name=snapshot::"
-          find lts/ -type f | sort --reverse --version-sort | head -n 1
+          find ../lts/ -type f | sort --reverse --version-sort | head -n 1
+
+      - name: Setup
+        run: |
+          m4 -D RESOLVER=${{ steps.prep.outputs.snapshot }} \
+            < stack.yaml.in > stack.yaml
```
Comment (pbrisbin) [inline .github/workflows/ci.yml:65]: Why did this become necessary?

**Review by pbrisbin — COMMENTED** [review]
Just curious why there is more here than the simple rename.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mjgpy3) [reply .github/workflows/ci.yml:65]: @pbrisbin the first commit was just a rename `lint` [failed](https://github.com/freckle/stackage-snapshots/runs/4715895384?check_suite_focus=true) looking for `stack.yaml`.

Comment (mjgpy3) [reply .github/workflows/ci.yml:65]: Perhaps the tool needs to be modified to not look for a stack.yaml if a resolver flag is given?

Comment (pbrisbin) [reply .github/workflows/ci.yml:65]: I think you just missed that L83 should now be `--path ${{ steps.prep.outputs.snapshot }}` and are misinterpreting a resulting error?

Comment (mjgpy3) [reply .github/workflows/ci.yml:65]: Oh I see. I guess I didn't realize that `stack.yaml` and these stack snapshots were interchangeable in that way.

**Comment by mjgpy3** [comment]
Okay that's looking a lot better now, thanks @pbrisbin. It looks like our last snapshot is behind but since it's immutable and I'll likely be fixing this in the next 18.21 snapshot, I'm gonna go ahead and merge this.

**Pull request merged**